template/{{.input_root_dir}}/.stacks_metadata.yml.tmpl

```diff
@@ -0,0 +1,11 @@
+# DO NOT MODIFY THIS AUTO-GENERATED FILE. 
+# MODIFYING THIS FILE MAY CAUSE YOUR STACK TO NOT FUNCTION PROPERLY
```

Comment:
Should we really claim this here?

Reply:
It is true, in the future we will use this UUID for other things too, so want to make sure no one modifies it